### PR TITLE
DIV-8274 | Fix certificate_processor not starting all enabled services

### PR DESCRIPTION
**Pull request by egov-joy**

### CODE REVIEW

File: backend/vaccination_api/cmd/certificate_processor/main.go [hunk 1]
```diff
@@ -126,33 +121,12 @@ func initializeRevokeCertificate() {
...
-	c, err := kafka.NewConsumer(&kafka.ConfigMap{
-		"bootstrap.servers":  config.Config.Kafka.BootstrapServers,
-		"group.id":           "revoke_cert",
-		"auto.offset.reset":  "earliest",
-		"enable.auto.commit": "false",
-	})
-
+	startRevokeCertificateErrorTopicProducer(producer)
+	c, err := createConsumer("revoke_cert", "earliest", "false")
+	err = c.SubscribeTopics([]string{config.Config.Kafka.RevokeCertTopic}, nil)
```
Comment (himeshr-egov) [inline backend/vaccination_api/cmd/certificate_processor/main.go:126]: what if you encounter error while creating consumer, should the err check not be present before subscribeTopic as well.?

File: backend/vaccination_api/cmd/certificate_processor/main.go [hunk 2]
```diff
@@ -66,36 +67,30 @@ type CertifyMessage struct {
...
-
-	c, err := kafka.NewConsumer(&kafka.ConfigMap{
-		"bootstrap.servers":  config.Config.Kafka.BootstrapServers,
-		"group.id":           "certificate_processor",
-		"auto.offset.reset":  "earliest",
-		"enable.auto.commit": "false",
-	})
-
+	c, err := createConsumer("certificate_processor", "earliest", "false")
+	err = c.SubscribeTopics([]string{config.Config.Kafka.CertifyTopic}, nil)
```
Comment (himeshr-egov) [inline backend/vaccination_api/cmd/certificate_processor/main.go:90]: what if you encounter error while creating consumer, should the err check not be present before subscribeTopic as well.?

Comment (egov-joy) [reply backend/vaccination_api/cmd/certificate_processor/main.go]: Fixed.

File: backend/vaccination_api/cmd/certificate_processor/main.go (hunk 1 above)
Comment (egov-joy) [reply backend/vaccination_api/cmd/certificate_processor/main.go]: Fixed.

**Review by himeshr-egov — APPROVED** [review]

**Pull request merged**